src/main/java/de/samply/share/client/util/connector/CentralSearchConnector.java

```diff
@@ -295,10 +323,8 @@ public void setLastUploadTimestamp(String timestamp) throws CentralSearchConnect
           SamplyShareUtils.addTrailingSlash(centralSearchUrl.getPath()) + PATH_UPLOAD_STATS);
 
       UploadStats uploadStats = new UploadStats(timestamp);
-      Serializer serializer = new Persister();
-      StringWriter writer = new StringWriter();
-      serializer.write(uploadStats, writer);
-      HttpEntity httpEntity = new StringEntity(writer.getBuffer().toString());
+      String stringUploadStats = serialize(uploadStats);
```

Comment:
I would not introduce a Variable for the serialized upload stats here. Just put the serialize call into the StringEntity constructor call.